tests/integration/backends/test_tpp.py

```diff
@@ -1262,6 +1265,174 @@ def test_vaccinations(select_all):
     ]
 
 
+def sha256_digest(int_):
+    return hashlib.sha256(int_.to_bytes()).digest()
+
+
+def to_hex(bytes_):
+    return bytes_.hex().upper()
+
+
+@register_test_for(tpp.wl_clockstops_raw)
+def test_wl_clockstops_raw(select_all):
+    results = select_all(
+        Patient(Patient_ID=1),
+        WL_ClockStops(
+            Patient_ID=1,
+            ACTIVITY_TREATMENT_FUNCTION_CODE="110",
+            PRIORITY_TYPE_CODE="1",
+            PSEUDO_ORGANISATION_CODE_PATIENT_PATHWAY_IDENTIFIER_ISSUER=sha256_digest(1),
+            PSEUDO_PATIENT_PATHWAY_IDENTIFIER=sha256_digest(1),
+            Pseudo_Referral_Identifier=sha256_digest(1),
+            Referral_Request_Received_Date="01-02-2023",
+            REFERRAL_TO_TREATMENT_PERIOD_END_DATE="03-04-2025",
+            REFERRAL_TO_TREATMENT_PERIOD_START_DATE="02-03-2024",
+            SOURCE_OF_REFERRAL_FOR_OUTPATIENTS="",
+            Waiting_List_Type="ORTT",
+            Week_Ending_Date="03-03-2024",
```

Comment:
The data in the db is in YYYY-MM-DD format; is it DD-MM-YYYY here deliberately, to check that the raw table just produces whatever string is there? If so, could there maybe be a comment to say so?

Reply:
This is an error on my part. I didn't have DB access when I wrote the tests, so I relied on the codebook. When I regained DB access, I checked the tables and updated the tests, but missed this test. Thanks for spotting it!